test/e2e-v2/cases/istio/als/e2e.yaml

```diff
@@ -56,6 +56,7 @@ setup:
           --set oap.env.SW_ENVOY_METRIC_ALS_HTTP_ANALYSIS=$ALS_ANALYZER \
           --set oap.env.SW_ENVOY_METRIC_ALS_TCP_ANALYSIS=$ALS_ANALYZER \
           --set oap.env.K8S_SERVICE_NAME_RULE='e2e::${service.metadata.name}' \
+          --set oap.env.SW_HEALTH_CHECKER=default \
```

Comment:
I think we should enable the healthchecker by default since the web app depends on it, what do you think? @wu-sheng